poison/callbacks.go

```diff
@@ -38,9 +39,9 @@ type StopCallback struct{}
 
 // Call exists service with log
 func (*StopCallback) Call() error {
-	log.Warningln("detected poison record, exit")
+	log.WithField(logging.FieldKeyEventCode, logging.EventCodePoisonRecordDetectionMessage).Warningln("Detected poison record, exit")
 	os.Exit(1)
-	log.Errorln("executed code after os.Exit")
+	log.WithField(logging.FieldKeyEventCode, logging.EventCodePoisonRecordDetectionMessage).Errorln("executed code after os.Exit")
```

Comment:
```suggestion
	log.WithField(logging.FieldKeyEventCode, logging.EventCodePoisonRecordDetectionMessage).Errorln("Executed code after os.Exit")
```

should it also start from the capital letter?

Reply:
no, this code should not be executed because after `os.Exit`. It's just honeypot that we check in our integration tests that it's not executed and service stopped.

Reply:
i see